onvm/onvm_mgr/onvm_init.c

```diff
@@ -329,8 +329,11 @@ init_nf_info_pool(void)
  */
 static int
 init_port(uint8_t port_num) {
-        const uint16_t rx_rings = ONVM_NUM_RX_THREADS, tx_rings = MAX_NFS;
+        const uint16_t rx_rings = ONVM_NUM_RX_THREADS;
         const uint16_t rx_ring_size = RTE_MP_RX_DESC_DEFAULT;
+        /* Currently, we reserve 1 core for stats thread, 1 core for 1 rx thread,
```

Comment:
Specify that we reserve the number of rings to be the same as tx threads in the manager. Perhaps just saying that this number of tx_rings matches the number of tx threads is better than giving a breakdown of cores here.

Reply:
I agree with the macro name change to ONVM_NUM_MGR_AUX_THREADS and the comment modification proposed by @koolzz .  Do I need to modify these and do a submission again? I don't know if koolzz has already fixed this or not. If not, I will do another submission.